apps/wallet/src/ui/app/components/accounts/AccountListItem.tsx

```diff
@@ -15,9 +15,10 @@ type AccountListItemProps = {
 	account: SerializedUIAccount;
 	editable?: boolean;
 	selected?: boolean;
+	showLocked?: boolean;
 };
 
-export function AccountListItem({ account, editable }: AccountListItemProps) {
+export function AccountListItem({ account, editable, showLocked = true }: AccountListItemProps) {
```

Comment:
we should default to false here
```
showLocked = false
```

to avoid having props like `showLocked={false}`